crates/sqlbuiltins/src/functions/scalars/mod.rs

```diff
@@ -0,0 +1,71 @@
+// we make use of the document! macro to generate the documentation for the builtin functions.
+// specifically the `stringify!` macro is used to get the name of the function.
+// `Abs` would otherwise be `Abs` instead of `abs`. and so on.
+#![allow(non_camel_case_types)]
+
+pub mod df_scalars;
+pub mod kdl;
+pub mod postgres;
+use crate::{
+    document,
+    functions::{BuiltinFunction, BuiltinScalarUDF, ConstBuiltinFunction},
+};
+use datafusion::logical_expr::BuiltinScalarFunction;
+pub use df_scalars::*;
```

Comment:
Hot take...

I prefer to just * export the entire module for things like this.  

IMO, the sub modules of `scalars::` are irrelevant. They are all providing scalar functions, the submodules are just used to keep file size smaller and to logically organize it internally. They're meant to all be part of one cohesive `scalars` module. This is also why I use the `use super::*` syntax for the submodules here.  Subjectively, it's easier to keep the code organized this way instead of one 3k LOC `scalars` file. 

Everyone has their own preferences on code organization, and i find this works best for me.

Reply:
spicy. 

I find these imports pretty confusing as it's not clear what the right way to reference anything because the symbols are accessible in different ways, the editing experience within files has pretty confusing completion options (which makes it harder to get involved in the codebase), and I enjoy the way that files/modules function as namespaces (which the glob imports erode); but also I don't really mind long files (if a file is more than 200 lines I'm probably using grep anyway so the difference between 300 lines and 3000 lines is immaterial.)

In any case, I think it's probably worthwhile that we have some guide/standard for the style and organization that isn't "one person's preference on any given day" and hopefully that's a set of things that we can have auto formatters/lints deal with so we don't spend time talking about it.